packages/connection/src/Manager.php

```diff
@@ -28,6 +29,155 @@ class Manager implements Manager_Interface {
 	 */
 	protected $secret_callable;
 
+	/**
+	 * A copy of the raw POST data for signature verification purposes.
+	 *
+	 * @var String
+	 */
+	protected $raw_post_data;
+
+	/**
+	 * Verification data needs to be stored to properly verify everything.
+	 *
+	 * @var Object
+	 */
+	private $xmlrpc_verification = null;
+
+	/**
+	 * Creates an instance of the connection manager.
+	 */
+	public function __construct() {
```

Comment:
Do we need a constructor if it doesn't do anything? Seems like we could remove it.